modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -861,6 +875,9 @@ TaskPriority getBreakingTaskPriority(Arguments options) {
             if (options.getDirectory() != null) {
                 skippedTasks.add("--datadir");
             }
+            if (options.getPort() != null) {
```

Comment:
Please add some tests that use the port option in the `CliTest.testCheckArguments` as well.